text/0000-api-doc-frontpage-styleguide.md

```diff
@@ -0,0 +1,174 @@
+- Feature Name: api_doc_frontpage_styleguide
+- Start Date: 2016-07-25
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+This RFC is a companion to the [API doc guidelines proposal]
+(https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md).
+In this RFC, we focus on the format and style of the "front page" of an API.  The goal of this 
```

Comment:
You mean front page of documentation as generated by rustdoc? “API” is a function or a set of functions exposed to other programmers and the acronym is in no way related to presentation.